src/lib/promise.js

```diff
@@ -0,0 +1,18 @@
+//https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html
+export const makeCancelable = promise => {
```

Comment:
maybe add a test for this helper?

Reply:
will use the one from cozy-client ;)

Reply:
@y-lohse : https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/utils.js#L9 ;)